pom.xml

```diff
@@ -98,6 +98,11 @@
       <version>3.5.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>mod-configuration-client</artifactId>
+      <version>4.0.4-SNAPSHOT</version>
```

Comment:
Sorry, there was no mod-configuration-client with a version 4.0.3 release.
It has been released yesterday so that you can use a non-SNAPSHOT version now.

Reply:
fixed